Jenkinsfile

```diff
@@ -140,6 +143,16 @@ pipeline {
                     }
                     publish()
                 }
+                script {
+                    if (env.version != "master") {
```

Comment:
I don't see what is the purpose of this ... do you want to publish a new docker image for every branch ?

Reply:
I don't want to publish an image when version is UNRELEASED. A few lines up we have:
```
 if (env.version == "UNRELEASED") {
    env.version = "master"
}
```